### PR TITLE
added navbar component

### PR DESCRIPTION
**Pull request by lannisterrr**
recording

Added Nabar Component with Input and Badge.

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.main-head {
+  background: var(--complimentary-bg-color);
+  color: var(--secondary-color);
+  /* position: sticky;
```
Comment (kedark152) [inline components/navbar/navbar.css:6]: Remove Commented code.

**Review by kedark152 — CHANGES_REQUESTED** [review]
Looks Good.

File: components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.nav-searchBar {
+  width: 300px;
+  padding-left: 4rem;
+  border: none;
+}
+
+.nav-searchBar:focus {
+  outline: none;
```
Comment (junaidshaikh-js) [inline components/navbar/navbar.css]: An element should be focusable, this will make components 
inaccessible to keyboard users.

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: components/navbar/navbar.css (hunk 2 above)
Comment (lannisterrr) [reply components/navbar/navbar.css]: Thanks for pointing that out , I will add a custom focus color to fix that.

**Review by ashishnnnnn — APPROVED** [review]
Look good to merge

**Review by kedark152 — APPROVED** [review]
Looks Good.

**Pull request merged**